pkg/cmd/taskrun/logs.go

```diff
@@ -39,6 +41,45 @@ type LogOptions struct {
 	AllSteps bool
 }
 
+func logCommand(p cli.Params) *cobra.Command {
+	opts := LogOptions{}
+	eg := `
+	# show the logs of TaskRun named "foo" from the namesspace "bar"
+	  tkn taskrun logs foo -n bar
+	 `
+	c := &cobra.Command{
+		Use:          "logs",
+		Short:        "Show taskruns logs",
+		Example:      eg,
+		SilenceUsage: true,
+		Args:         cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+
+			cs, err := p.Clients()
+			if err != nil {
+				return err
+			}
+
+			trl := &TaskRunLogs{
+				Run:     args[0],
+				Ns:      p.Namespace(),
+				Clients: cs,
+			}
+
+			s := logs.Streams{
```

Comment:
I am thinking if we should add a  `params.Stream()` which get auto initialised to `cmd.OutOrStdOut, and cmd.ErrOrStderr()`  which can be faked in tests

Reply:
Same as above, it will be useful in other places. It will be resolved together in the upcoming iteration.